serialization/mappack.py

```diff
@@ -39,6 +39,7 @@ def unpack_map(fname):
 
     # Read the Magic number/version(?)
     if arcfile.read(8) != b'XBIN\x34\x12\x02\x00':
+        arcfile.close()
```

Comment:
Not sure why `arcfile` doesn't get created by the `open` context manager automatically.
It would be better if this was that case.
Ie.
```python
with open(fname, 'rb') as arcfile:
# do some stuff...
```

Reply:
Ah, that's cool! I didn't know about "with" until now! I'll rewrite the fix to just utilize "with".